pom.xml

```diff
@@ -184,7 +177,7 @@ under the License.
     <dependency>
       <groupId>org.apache.maven.shared</groupId>
       <artifactId>maven-dependency-tree</artifactId>
-      <version>3.2.1</version>
+      <version>3.3.0</version>
```

Comment:
We should drop it and use Resolver ... but can be done as separate issue

Reply:
Yes, i know, but that is a separate issue and I have no clue how to do that. I'd appreciate a PR for it.

Reply:
Done: https://issues.apache.org/jira/browse/MPIR-458